unicef/giga-dagster
# feat: add master_release_notification_emails

## What type of PR is this?

- `feat`: Commits that add a new feature

## Summary

Adds automated sending of to users who have had their country master data updated.

Email template can be found here https://github.com/unicef/giga-data-ingestion/pull/49

## How to test
0. Make sure you are first a part of the country you want to work with. You can do this through the `User management` tab or by manually giving yourself groups in Azure AD
1. Check out the following branch in `data-ingestion` https://github.com/unicef/giga-data-ingestion/pull/49
2. Go to `localhost:3001` ->`Deployment` -> `Reload All`
3. Recreate the `school_master__convert_gold_csv_to_deltatable_jobs`. Go to `Runs` -> Delete the run for your country
4. Turn on the `school_master_gold_csv_to_deltatable_sensor` and wait for it to evaluate
5. Materialize the `terminate_all_runs_job`
6. Terminate any runs that are currently `In Progress` and then wait for the `admin__terminate_all_runs` run to finish
7. Grab a file of the contry you chose in `updated_master_schema/master`
8. Modify some rows of the downloaded file and then place it in the corroesponding country folder in `updated_master_schemam/master_updates;`,. Make sure to rename it with a new timestamp so the original file does not get overwritten. Also make sure that the newset file has the latest `modified` timestamp
9. Run (Re-execute) the run related to your country
10. You should receive an email.

## Link to Jira/Asana/Airtable task (if applicable)

https://app.asana.com/0/1206270489214055/1206284876784200

## Implementation screenshot/screencap (if applicable)

_placeholder_
